lib/matplotlib/__init__.py

```diff
@@ -444,17 +444,19 @@ def _create_tmp_config_or_cache_dir():
 def _get_xdg_config_dir():
     """
     Return the XDG configuration directory, according to the `XDG
-    base directory spec
-    <http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_.
+    base directory spec:
```

Comment:
Keep the markup; below too.

Reply:
It would be too long for the character limit. I didn't want to add a flake8 ignore for that. Since this is a private function, which is not rendered in the docs, I figured it's ok to remove the markup.

Reply:
ok but then at least remove the backquote before XDG, so that the thing stays correct markup (even if not rendered.
below too.

Reply:
sure.